src/ansys/aedt/core/modeler/advanced_cad/oms.py

```diff
@@ -25,31 +25,30 @@
 import os
 import warnings
 
-from ansys.aedt.core.generic.general_methods import is_ironpython
 from ansys.aedt.core.generic.general_methods import pyaedt_function_handler
 from ansys.aedt.core.generic.general_methods import settings
 
 logger = settings.logger
-if not is_ironpython:
-    try:
-        import numpy as np
-
-    except ImportError:
-        warnings.warn("numpy package is needed")
-
-    try:
-        import pyvista as pv
-        import vtk
-    except ImportError:
-        warnings.warn("pyvista package is needed")
-
-    try:
-        import osmnx as ox
-        import srtm
-        import utm
-
-    except ImportError:
-        warnings.warn("OpenStreetMap Reader requires utm, osmnx and srtm extra packages")
+
+try:
+    import numpy as np
+
+except ImportError:
+    warnings.warn("numpy package is needed")
```

Comment:
```suggestion
try:
    import numpy as np
except ImportError:
    warnings.warn("
        "The NumPy module is required to use the OpenStreetMap Reader.\n"
        "Install with \n\npip install numpy"
    )
```
Just taking this review as an opportunity to apply snake case when I see it needed :)